### ISSUE TITLE
cross validation

### ISSUE THREAD

**Issue by MEHWISHSABA** [amueller/scipy_2015_sklearn_tutorial#62]
i m doing cross validation .the following code as shown.what;s error in this because it gives an error plz solve this issue..

from sklearn.cross_validation import KFold

def run_cv(X,y,clf_class,**kwargs):
    # Construct a kfolds object
    kf = KFold(len(y),n_folds=5,shuffle=True)
    y_pred = y.copy()
    
    # Iterate through folds
    for train_index, test_index in kf:
        X_train, X_test = X[train_index], X[test_index]
        y_train = y[train_index]
        # Initialize a classifier with key word arguments
        clf = clf_class(**kwargs)
        clf.fit(X_train,y_train)
        y_pred[test_index] = clf.predict(X_test)
    return y_pred

E:\Anaconda folder\lib\site-packages\sklearn\cross_validation.py:44: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.
  "This module will be removed in 0.20.", DeprecationWarning)
what should i do now???

**Comment by amueller** [comment]
It's not an error, it's a warning, and the code still works. To avoid the warning, use the updated version of the tutorial at https://github.com/amueller/scipy-2017-sklearn/

**Comment by MEHWISHSABA** [comment]
sir
updated version for what

**Comment by MEHWISHSABA** [comment]
i m doing this code at jupyter notebook

**Comment by MEHWISHSABA** [comment]
dear sir.
i need data description of the customer churn dataset this is the link of
dataset
 https://github.com/tarikd/Churn-Prediction/blob/master/churn.csv

i extremely need it .because i want to know why these fields(state, area
code,phone number ,churn) are removed in customer churn dataset .so that i
will be able to get the information about dataset.
so please sir send me data description as soon as possible. i will be very
thankful to you.

**Comment by MEHWISHSABA** [comment]
sir
is the link working and opening the github to you ,because its not working
to me now i dont know what happened to this site
but sir i need data dictionary of customer churn dataset . plz sir send me
as soon as possible i extremely need that
i need data dictionary earlier  i wrote data description sorry for that

**Comment by amueller** [comment]
yes, the link to the 2017 version works. Maybe check it out in github and not in your mail.
And I'm not sure what your churn data has to do with this issue.

**Comment by MEHWISHSABA** [comment]
sir my churn dataset will predict those customers who are likely to leave
company that's way i need data dictionary to know the description plz sir
send me data dictionary customer churn data set .this is dataset
